solutions/devsprint-matheus-reis-1/RealEstateAppTests/Components/LoadingViewTest.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LoadingViewTest.swift
+//  RealEstateAppTests
+//
+//  Created by Cora on 11/05/22.
+//
+
+import Foundation
+import XCTest
+import SnapshotTesting
+@testable import RealEstateApp
+
+class LoadingViewTest: XCTestCase {
+    
+    var sut =  LoadingView()
+    
+    override func setUpWithError() throws {
+       
+    }
+
+    func testDividerView() {
```

Comment:
O nome do teste ficou como `testDividerView`, imagino que tenha replicado o arquivo e faltou fazer essa alteração.
```suggestion
    func testLoadingView() {
```